consensus/src/round_manager.rs

```diff
@@ -531,7 +544,48 @@ impl RoundManager {
         );
 
         observe_block(proposal.timestamp_usecs(), BlockStage::SYNCED);
+        if self.decoupled_execution() && self.block_store.back_pressure() {
+            // In case of back pressure, we delay processing proposal. This is done by resending the
+            // same proposal to self after some time.
+            Ok(self
+                .resend_verified_proposal_to_self(
+                    proposal,
+                    BACK_PRESSURE_POLLING_INTERVAL_MS,
+                    BACK_PRESSURE_PROPOSAL_TIMEOUT_MS,
```

Comment:
this probably shouldn't be a constant, probably round deadline minus some constant

Reply:
How about something like `round_initial_timeout_config * 0.9 ` ?

Reply:
sounds good

Reply:
Actually, I am leaving this value to equal to `round_initial_timeout` for now. We can tune it further if needed.